### DISCUSSION TITLE
Converting latex theorems into markdown containers,

### DISCUSSION THREAD

**Discussion by schcs** [jgm/pandoc#10239] [Q&A]
I would like to convert some latex files to markdown so that they can be processed by Quarto. I would that theorem-like environments should be converted into custom containers. For example, 

```
\begin{theorem}
My theorem.
\end{theorem}
```

should be converted to 

```
:::{#thm-}
My theorem.
:::
```
(or something similar that I can easy process with some extra program. Howver, running pandoc only converts it into 
```
My theorem.
```
What am I doing wrong?

**Comment by massifrg** [comment]
The latest version actually gives this:

```markdown
::: theorem
My theorem.
:::
```

as you can check at [Try pandoc!](https://pandoc.org/try/)

**Reply by schcs** [reply]
Yes, you are right. I had version 2.9 installed. Thank you.
